docs/index.txt

```diff
@@ -9,7 +9,24 @@ Crate provides a type 4 JDBC driver. The driver is written in Pure Java, and
 communicates in the database system's own network protocol. Because of this,
 the driver is platform independent.
 
-.. note:: This JDBC driver requires Crate 0.38.0 or newer!
+Compatibility
```

Comment:
can you move it here https://github.com/crate/crate-jdbc/blob/6a7b1f93f8e4b2f3c88651929d0713a101c784a1/docs/index.txt#L220
before releasing